nimbus/src/lib.rs

```diff
@@ -292,6 +307,7 @@ impl NimbusClient {
 
 #[derive(Debug, Clone)]
 pub struct EnrolledExperiment {
+    pub feature_id: String,
```

Comment:
@travis79 may have a better idea: I thought the mapping from `Experiment` to `EnrolledExperiment` was `at most one`.

Is `EnrolledExperiment` a feature within an experiment, or should this be an array?

Reply:
I think that this should be an array, as per discussion in many places, experiments could potentially have multiple feature ids, basically this should be a duplicate of the array in the `Experiment` object.

Reply:
So is there a need to add `feature_id` in `EnrolledExperiment` at all? Either it has all the feature ids or doesn't need any of them?

I don't think this should hold up this PR unless the answer is obvious, @travis79 .

Reply:
Eh, I don't see a need to have the feature ids in the Android Component, now that you mention it, unless we need to record that in telemetry somewhere, but that should be able to be referenced outside of telemetry right now.  Happy to see it omitted here, unless we determine there is a need for it before we migrate the Glean integration to Rust.

Reply:
We have to have at least one `feature_id` there, because the DatabaseCache uses EnrolledExperiments internally, and if they don't expose at least one `feature`, we can't build up the HashTable we need to search on.  I'll switch it to an array.